eng/pipelines/smoke-tests.yml

```diff
@@ -39,7 +41,39 @@ jobs:
           packageType: sdk
           version: "$(DotNetCoreSDKVersion)"
 
-      - script: dotnet run -p ./common/SmokeTests/SmokeTest/SmokeTest.csproj --framework $(TestTargetFramework)
+      - pwsh: Register-PackageSource -Name NightlyFeed -Location '$(BlobFeedUrl)' -ProviderName Nuget
+        displayName: Register nightly feed package source
+
+      - task: PowerShell@2
+        inputs:
+          targetType: filePath
+          filePath: ./common/SmokeTests/SmokeTest/Update-Dependencies.ps1
+          arguments: -Framework $(TestTargetFramework)
+          workingDirectory: common/SmokeTests/SmokeTest
+          pwsh: true
+        displayName: Use latest dev feed package versions
+
+      - task: PowerShell@2
+        inputs:
+          targetType: filePath
+          filePath: ./common/SmokeTests/SmokeTest/Test-DependencyDevVersions.ps1
+          workingDirectory: common/SmokeTests/SmokeTest
+          pwsh: true
+        continueOnError: true
+        displayName: Validate Package Versions
+
+      - pwsh: dotnet restore ./common/SmokeTests/SmokeTest/SmokeTest.csproj
+        displayName: dotnet restore
+
+      - pwsh: >
```

Comment:
Do you plan to keep this? Or did you want to just dump the project file or diff of the project file?